hiss/application/views.py

```diff
@@ -47,3 +47,6 @@ def get_object(self, queryset=None) -> Application:
         if app.user != self.request.user:
             raise PermissionDenied("You don't have permission to view this application")
         return app
+
+    def get_success_url(self, **kwargs):
+        return reverse_lazy("status")
```

Comment:
Because the success URL isn't dynamic, this can be just included as a member of the class. The [`FormMixin`](https://docs.djangoproject.com/en/2.2/ref/class-based-views/mixins-editing/#django.views.generic.edit.FormMixin.success_url) class (which [`UpdateView` inherits from](https://docs.djangoproject.com/en/2.2/ref/class-based-views/mixins-editing/#django.views.generic.edit.FormMixin.success_url)) has the `success_url` parameter, which can (and often should) be used in cases like this.

Delete this function, and add `success_url` to the class instead.

```python
        success_url = reverse_lazy("status")
```